complex/ft_test.c

```diff
@@ -310,6 +312,25 @@ void ft_next_iov_cnt(struct ft_xcontrol *ctrl, size_t max_iov_cnt)
 		ctrl->iov_iter = 0;
 }
 
+int ft_next_ctx(struct ft_xcontrol *ctrl, uint64_t comp)
+{
+	int ret;
+
+	ctrl->curr_ctx++;
+	if (ctrl->curr_ctx < ctrl->max_credits)
+		return 0;
+
+	while (ctrl->credits < ctrl->max_credits && comp == FI_SEND) {
+		ret = ft_comp_tx(FT_COMP_TO);
+		if (ret)
+			return ret;
+	}
```

Comment:
I see. Yes, out of order completions would not allow us to reuse the context. Thanks!